app/helpers/template_helpers.py

```diff
@@ -112,10 +112,11 @@ def page_header_context(self) -> dict[str, Union[bool, str, LazyString]]:
         context: dict[str, Union[bool, str, LazyString]] = {
             "orgLogo": f"{self._survey_config.page_header_logo}",
             "orgLogoAlt": f"{self._survey_config.page_header_logo_alt}",
+            "title": self._survey_title
+            if self._survey_title and cookie_session.get("theme")
```

Comment:
`cookie_session.get("theme")` is available as `self._survey_type`.

Reply:
Using `self._survey_type` now.